web/src/lib/managers/timeline-manager/internal/load-support.svelte.ts

```diff
@@ -38,6 +38,9 @@ export async function loadFromTimeBuckets(
       },
       { signal },
     );
+    if (!albumAssets) {
+      return;
+    }
```

Comment:
When can this be `null`?

Reply:
Oh, if the service worker cancels the request in-flight?

Reply:
Not the service worker in this case, but yes. CancellableTask is used for canceling time buckets.